gerrit-entrypoint.sh

```diff
@@ -54,6 +54,34 @@ if [ "$1" = "/gerrit-start.sh" ]; then
     echo
   done
 
+#Replication config
+if [ -n "${REPLICATION_REMOTES}" ]; then
+cat <<EOF > $GERRIT_SITE/etc/replication.config
+[gerrit]
+	autoReload = true
+EOF
+fi
+
+for r in $REPLICATION_REMOTES; do
+  REMOTE=`eval $(echo echo \\$$(echo "${r}_REMOTE"| awk '{print toupper($0)}'))`
+  PASSWD=`eval $(echo echo \\$$(echo "${r}_PASSWORD"| awk '{print toupper($0)}'))`
+  MIRROR=`eval $(echo echo \\$$(echo "${r}_MIRROR"| awk '{print toupper($0)}'))`
+  PROJECTS=`eval $(echo echo \\$$(echo "${r}_PROJECTS"| awk '{print toupper($0)}'))`
+  REPLICATE_ON_STARTUP=`eval $(echo echo \\$$(echo "${r}_REPLICATE_ON_STARTUP"| awk '{print toupper($0)}'))`
+
+  [ -z "${PASSWD}" ] || set_secure_config remote.$r.password "${PASSWD}"
+
+cat <<EOF >> $GERRIT_SITE/etc/replication.config
+[remote "$r"]
+	url = $REMOTE
+	push = +refs/tags/*:refs/tags/*
+	push = +refs/heads/*:refs/heads/*
+	mirror = $MIRROR
+	projects = $PROJECTS
+	replicateOnStartup = $REPLICATE_ON_STARTUP
+EOF
+done
+
```

Comment:
To support replication.config is a great feature. However I'd like to separate and review this feature in another PR.